src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -55,18 +78,33 @@ const CreateOrEditNote = ({
     }
   })
 
+  const userPhoto = photo
+    ? `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`
+    : undefined
+  const isNameValid = firstName && lastName
```

Comment:
Please wrap is with `Boolean` constructor because this line will not yield a boolean if you think about it

Reply:
Fixed